src/sentry/api/decorators.py

```diff
@@ -17,6 +17,11 @@ def is_considered_sudo(request):
     )
 
 
+def has_verified_email(request):
+    user = request.user
+    return request.user.get_verified_emails().filter(email=user.email).exists()
```

Comment:
to make this generalizable need to think about:
- what should happen with api keys and tokens?
- should superuser bypass this check?
- should this check anything else?

Reply:
- within the API the same check is fine - it will just need to return a valid API response (an error)
- it should never be bypassed
- i think "is any email verified" is fine, but I haven't put deeper thought into it. for outbound emails we'd like to also move to only sending them (for non-verification) if the given email is verified

Reply:
got it. this should be fine as is for apikeys and superuser then. and changed from primary to any verified email.